app/views/directives/notifications/header.html

```diff
@@ -0,0 +1,29 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6">
+      <strong>{{notificationGroup.heading}}</strong>
+    </div>
+    <div class="col-md-6 text-right small">
+      <a
+        ng-init="allEventsUrl = 'project/' + $ctrl.customScope.projectName + '/browse/events'"
```

Comment:
You can just make this an `ng-href` and have an `ng-click` handler as well that only closes the drawer. Otherwise right-click won't work.